vulnerable_people_form/integrations/govuk_notify_client.py

```diff
@@ -1,110 +1,68 @@
-]
-
-SMS_CONTACT_GP_PARAGRAPHS = [
-    "Contact your doctor as soon as possible so they can confirm to us you’re classed as clinically extremely vulnerable. You may not get the support you need if you do not contact them.",
-    "After that we’ll check that you’re eligible, and you’ll start getting support if you asked for it.",
-]
-
-SMS_NO_CONTACT_GP_PARAGRAPHS = [
-    "Contact your doctor as soon as possible so they can confirm to us you’re classed as clinically extremely vulnerable. You may not get the support you need if you do not contact them.",
-    "After that we’ll check that you’re eligible, and you’ll start getting support if you asked for it.",
-]
-
-NO_GP_CONTACT_SMS_MESSAGE = "\n\n".join(
-    SMS_GREETING_PARAGRAPHS + SMS_NO_CONTACT_GP_PARAGRAPHS + SMS_CLOSING_PARAGRAPHS
-)
-GP_CONTACT_SMS_MESSAGE = "\n\n".join(
-    SMS_GREETING_PARAGRAPHS + SMS_CONTACT_GP_PARAGRAPHS + SMS_CLOSING_PARAGRAPHS
-)
+def _try_and_report_exception_to_sentry(fn, *args, **kwargs):
+    try:
+        fn(*args, **kwargs)
+    except Exception as e:
+        sentry_sdk.capture_exception(e)
 
 
-def send_confirmation_sms(phone_number, first_name, last_name, need_to_contact_gp):
-    message = (
-        GP_CONTACT_SMS_MESSAGE if need_to_contact_gp else NO_GP_CONTACT_SMS_MESSAGE
-    ).format(first_name=first_name, last_name=last_name)
+def send_spl_failure_sms(
+    reference_number, phone_number, first_name, last_name, told_to_shield
+):
+    message = "\n\n".join(
+        (
+            f"Hello {first_name} %{last_name} - your registration for the shielding service is %{reference_number}.",
+            *(
+                (
+                    "You do not need to do anything else if you've been told to shield by the NHS or your doctor and they've put you on the NHS list of people who should be shielding.",
+                    "We’ll check your details, then contact you to confirm whether you’re eligible for support. This can take up to 2 weeks.",
+                )
+                if told_to_shield
+                else (
+                    "Contact your GP or hospital clinician as soon as possible so they can put you on the NHS list of people who should be shielding. You may not get the support you need if you do not contact them.",
+                    "We’ll check your details, then contact you to confirm whether you’re eligible for support. This can take up to 2 weeks from when your GP or hospital clinician puts you on the NHS list of people who should be shielding.",
+                ),
+            ),
+            "Contact your local authority if you need support urgently and cannot rely on family, friends or neighbours: www.gov.uk/coronavirus-local-help",
+        )
+    )
     get_notifications_client().send_sms_notification(
         phone_number=phone_number,
         template_id=current_app.config["GOVUK_NOTIFY_SMS_TEMPLATE_ID"],
```

Comment:
i will provide new template id's for this there will be 6 in total 3 for each type sms, email and letter and whether there is a match or not

Reply:
i have added all the env vars into master now so if you do a rebase then you should get all the template ids @danwilliams